src/components/OverlayManager/Manager.tsx

```diff
@@ -44,4 +44,4 @@ const Manager: React.FC = () => (
   </OverlayContext.Consumer>
 );
 
-export default Manager;
+export default OverlayManager;
```

Comment:
Well my intention was that since u r in `OverlayManager`, it is obvious that the `Manger` is the overlay one. 
But that change is fine with me.

Reply:
@piotrgrundas Yeah it makes sense! But I found myself confused several time by the difference between the names in devtools tree and files :)